Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
I took this Procfile from here:
https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/Procfile

This file needs to be added for production deployments with react_on_rails: https://github.com/shakacode/react_on_rails/blob/master/docs/additional-reading/heroku-deployment.md#heroku-deployment